nd/frontend/src/components/NavigationBar/NavigationBar.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import { useNavigate, Link } from "react-router-dom";
+import styles from "./NavigationBar.module.scss";
+import logo from "../../assets/logoMain.svg";
+import { ROUTES } from "../../router/consts";
+
+const NavigationBar = ({ user }) => {
+  const navigate = useNavigate();
+
+  const links = [
```

Comment:
kintamuosius/funkcijas kurie nenaudoja state'o reiktų laikyti virš  komponento (outside of component), nes dabar kiekvienakart persikrovus komponentui sukuriami tie patys duomenys ir eikvojami resursai.